main.py

```
@@ -61,27 +97,10 @@ def read_and_encode_file(file_path, encode=True):
-        else:
-            logger.error(f"An error occurred: {str(e)}")
-            raise e
-
-    logger.info(f"Pushed to Github")
+    # Update or create files on GitHub
+    save_and_update_github(all_years_csv_path, repo, "data/daily-treasury-rates.csv")
+    save_and_update_github(all_years_pickle_path, repo, "data/daily-treasury-rates.pkl")
+    save_and_update_github(csv_path, repo, f"data/{current_year}-daily-treasury-rates.csv")
+    save_and_update_github(pickle_path, repo, f"data/{current_year}-daily-treasury-rates.pkl")
 
+    logger.info(f"All files pushed to Github")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)